clang/test/SemaCXX/overloaded-operator.cpp

```diff
@@ -598,3 +600,27 @@ namespace B {
 }
 void g(B::X x) { A::f(x); }
 }
+
+namespace static_operator {
+#if __cplusplus >= 201703L
```

Comment:
Why C++17?

Reply:
No way to modify things in a C++11/14 constexpr function (`++x` is not a constant expression), so no way to make a positive test that the expression was evaluated. Though I guess I should have done `(non_constexpr_fn(), X())()` and see if that failed.

Reply:
> No way to modify things in a C++11/14 constexpr function (`++x` is not a constant expression), so no way to make a positive test that the expression was evaluated. Though I guess I should have done `(non_constexpr_fn(), X())()` and see if that failed.

Oh I see. It's inconvenient that runtime results can't be easily asserted in this test file.

I think the related changes of constexpr were made in C++14 ([N3652](https://wg21.link/n3652)).